src/conditionals/third-party/wordproof-integration-active-conditional.php

```diff
@@ -3,35 +3,19 @@
 namespace Yoast\WP\SEO\Conditionals\Third_Party;
 
 use Yoast\WP\SEO\Conditionals\Conditional;
-use Yoast\WP\SEO\Helpers\Wordproof_Helper;
 
 /**
  * Conditional that is met when the WordProof integration is toggled on.
  */
 class Wordproof_Integration_Active_Conditional implements Conditional {
 
 	/**
-	 * The WordProof helper.
+	 * Returns whether or not the WordProof Timestamp integration is active.
 	 *
-	 * @var Wordproof_Helper
-	 */
-	private $wordproof;
-
-	/**
-	 * WordProof integration active constructor.
-	 *
-	 * @param Wordproof_Helper $wordproof The options helper.
-	 */
-	public function __construct( Wordproof_Helper $wordproof ) {
-		$this->wordproof = $wordproof;
-	}
-
-	/**
-	 * Returns whether or not the WordProof Timestamp plugin is active.
-	 *
-	 * @return bool Whether or not the WordProof Timestamp plugin is active.
+	 * @return bool Whether or not the WordProof Timestamp integration is active.
 	 */
 	public function is_met() {
-		return $this->wordproof->integration_is_active();
+		$options = \get_option( 'wpseo' );
+		return $options['wordproof_integration_active'];
```

Comment:
There was a discussion on slack about using WPSEO options in the initialiser that triggers all the upgrade routines. After discussion we reached the conclusion is that we should use `get_option` in this case, also since this code will be deprecated soon.